openlibrary/macros/databarWork.html

```diff
@@ -123,7 +123,7 @@ <h3 class="closed">
     </div>
     </div>
 
-$if isbn or oclc_numbers or 'lendinglibrary' in collection or library:
+$if (isbn or oclc_numbers or 'lendinglibrary' in collection or library) and not any_private_collections(collection):
```

Comment:
If the comment above is correct, it can be solved by passing in the single collection as `[collection]` or adjusting the `any_private_collections` call to check whether `collections` is a `list` or a collection.